components/blocks/SiteFooter/SiteFooter.vue

```diff
@@ -1,22 +1,36 @@
 <template>
   <div class="site-footer">
     <p class="site-footer__copyright">Copyright 2021</p>
+    <LanguageSelector
+      class="site-footer__language-selector"
+      data-testid="language-selector"
+    />
```

Comment:
I suppose that'd be a question for the design team, since this wasn't considered with the figma mockups. I'm open to the idea of moving it, but looking at other sites, most of them included this functionality in the footer, so I put it there to start.

Reply:
Aight I'll ask in the Discord, but I'll resolve this for now. If they'd like this to be changed, we can always open up another PR. No point delaying this one just for the sake of styling.